### ISSUE TITLE
Undefined symbols for architecture arm64:

### ISSUE THREAD

**Issue by AdamBCo** [Wondermall/Doppelganger#5]
I keep getting the following error message when using Doppelgänger in my project:

Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_WMLArrayDiffUtility", referenced from:
      objc-class-ref in PhotoTableViewController.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

What should I be doing to fix this issue?

Thanks,
- Adam

**Comment by zats** [comment]
how did you install the library? through cocoapods or as a subproject?

**Comment by AdamBCo** [comment]
Hey @zats thanks for the reply.

I installed Doppelgänger using Cocoapods.

**Comment by zats** [comment]
Does the sample project (`pod try`) working for you? Are you sure you are opening a workspace and not project?
Maybe you can share your project If nothing else works

**Comment by AdamBCo** [comment]
When using "pod try" i get the following error message:

Adams-MBP:Doppelganger-master-2 adamcooper$ pod try "Doppelganger"
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
Updating spec repositories

I downloaded the sample project via Github, but the workspace isn't opening properly. It keeps telling me that the podfile is missing:

Errors when running pod Install:

[!] No `Podfile' found in the project directory.

Errors in XCode:
diff: /../Podfile.lock: No such file or directory
diff: /Manifest.lock: No such file or directory
error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.

**Comment by zats** [comment]
Hey, you are right, demo project wasn't compiling 
apparently podspec file format changed slightly.
Please give the sample project another try and let me know if that works

**Comment by AdamBCo** [comment]
Yeah I'm still having issues with running both the control and demo project.

**Comment by zats** [comment]
is it possible to ask you to share barebones project setup so I can play with it?
